app/app.go

```diff
@@ -1177,6 +1202,7 @@ func (app *Evmos) setupUpgradeHandlers() {
 		// revenue module is deprecated in v19
 		storeUpgrades = &storetypes.StoreUpgrades{
 			Deleted: []string{"revenue"},
```

Comment:
```suggestion
```

Reply:
@fedekunze the removal of this module was not released yet (on mainnet).
This module is removed on v19